Makefile

```diff
@@ -152,3 +152,27 @@ runserver: ## Run the Django application locally
 
 .PHONY: dev-init
 dev-init: load-design-system-templates collectstatic makemigrations migrate createsuperuser ## Run the pre-run setup scripts
+
+.PHONY: functional-tests-up
+functional-tests-up:  ## Start the functional tests docker compose dependencies
+	docker compose -f functional_tests/docker-compose.yml up -d
+
+.PHONY: functional-tests-up-dev
+functional-tests-up-dev:  ## Start the functional tests docker compose dependencies and dev app
```

Comment:
Nit: `dev-up` to be consistent with the existing ones.

Reply:
My reasoning for doing it this way round was making the targets more discoverable and consistent with shell auto-complete (e.g. if you hit tab on `make functionial-tests-up` then it can suggest this as a variant), but I can change them for consistency if we prefer the convention on the other new targets.

Reply:
I don't disagree, there was a [convo](https://github.com/ONSdigital/dis-wagtail/pull/38#discussion_r1857286036) on this PR about it. I don't mind either if we are consistent.

Reply:
I've renamed to be consistent for now 👍